test/urban_disco/core_test.clj

```diff
@@ -3,5 +3,5 @@
             [urban-disco.core :refer :all]))
 
 (deftest a-test
-  (testing "FIXME, I fail."
-    (is (= 0 1))))
+  (testing "Passing test"
```

Comment:
You can probably either remove this file entirely, or write a test for something in the core ns.